### PR TITLE
Add method to retrieve all members from projects (including inherited)

### PR DESCRIPTION
**Pull request by sluc23**
Implements this API method: https://docs.gitlab.com/ee/api/members.html#list-all-members-of-a-group-or-project-including-inherited-members

### CODE REVIEW

**Comment by sluc23** [comment]
test added

**Comment by bradjones1** [comment]
@m1guelpf @sluc23 I was looking for this functionality and ran across this merge and resulting revert; what's the current status/cause for the revert?

**Comment by m1guelpf** [comment]
@bradjones1 See https://github.com/m4tthumphrey/php-gitlab-api/pull/418

File: lib/Gitlab/Api/Projects.php [hunk 1]
```diff
@@ -292,6 +292,37 @@ public function members($project_id, $parameters = [])
...
+     * )
+     *
+     * @throws MissingOptionsException  If a required option is not provided
+     *
+     * @return mixed
+     */
+    public function membersAll($project_id, $parameters = [])
+    {
+        if (!is_array($parameters)) {
+            @trigger_error("Deprecated: String parameter of the members() function is deprecated.", E_USER_NOTICE);
```
Comment (GrahamCampbell) [inline lib/Gitlab/Api/Projects.php:309]: Why did you add new code that was already marked as deprecated?
